Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -414,6 +442,16 @@
   categories:
   - UplinkChemicals
 
+- type: listing
+  id: UplinkSyringePistol
+  name: uplink-syringe-pistol-name
+  description: uplink-syringe-pistol-desc
+  productEntity: SyringePistol
+  cost:
+    Telecrystal: 15
+  categories:
+  - UplinkChemicals
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)